### PR TITLE
Add note to documentation when creating a custom action

### PR DESCRIPTION
**Pull request by Mentos93**
I am targetting this branch, because it is an important note for when users want to create a custom action and render it directly into the template.

## Changelog

```markdown
### Added
- Added a note to the recipe_custom_action.rst file

```

## To do

- [ ] Update the tests
- [ ] Update the documentation
- [ ] Add an upgrade note

## Subject

recipe_custom_action.rst file to document a note for when a user want to render a custom action directly into a template.

### CODE REVIEW

**Comment by greg0ire** [comment]
Thanks for your contribution! I changed the base branch to 3.x ;)

File: Resources/doc/cookbook/recipe_custom_action.rst [hunk 1]
```diff
@@ -257,3 +257,7 @@ The full ``CarAdmin.php`` example looks like this:
             // ...
         }
     }
+
+.. note::
+
+    If you want to render a custom controller action in a template by using the render function in twig you need to add `_sonata_admin` as an attribute. For example; `{{ render(controller('AppBundle:XxxxCRUD:comment', { '_sonata_admin': 'sonata.admin.xxxx' })) }}`. This has to be done because the moment the rendering should happen no routing is involved yet, and then you will get an error 'There is no _sonata_admin defined for the controller AppBundle\Controller\XxxxCRUDController and the current route ''.'
```
Comment (greg0ire) [inline Resources/doc/cookbook/recipe_custom_action.rst]: Please open the file with vim, go on this line and type `gqq`, it should wrap the lines.

Comment (greg0ire) [inline Resources/doc/cookbook/recipe_custom_action.rst]: Also , this is rst, so you will need to use double backticks

```
``_sonata_admin``
```

Comment (greg0ire) [inline Resources/doc/cookbook/recipe_custom_action.rst]: And please replace "no routing is involved yet" with "the routing, which usually sets the value for this parameter, is not involved at all"

Comment (Mentos93) [inline Resources/doc/cookbook/recipe_custom_action.rst]: Ok, sorry. Some lack of experience I guess. Will fix this, and use branch 3.x instead.

Comment (greg0ire) [inline Resources/doc/cookbook/recipe_custom_action.rst]: Don't apologize, it's great that you're willing to contribute! Please fix the tiny problems that are left and it should be good to merge :)

**Comment by greg0ire** [comment]
Duplicate of #4201

**Pull request closed without merging**